apps/nextjs-website/src/lib/api.ts

```diff
@@ -8,6 +9,10 @@ export function getOverviewPaths(): readonly string[] {
   return overviews.map((overview) => overview.path);
 }
 
+export function getProducts(): readonly Product[] {
+  return products.map((product) => product);
```

Comment:
```suggestion
  return products;
```
at the moment we don't need to iterate over products